tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": ".",
+    "lib": ["dom", "es2017"],
+    "module": "commonjs",
+    "paths": {
+      "*": ["*"]
+    },
+    "sourceMap": false,
+    "strict": true,
+    "target": "es6",
+    "typeRoots": ["./node_modules/@types", "./index.d.ts"]
+  },
+  "include": ["./lib/**/*", "./test/**/*.ts"]
```

Comment:
should this be the module folder instead of the lib folder? we shouldn't need to build before running tests right?

Reply:
that makes sense!